service/history/execution/mutable_state_util.go

```diff
@@ -328,6 +328,7 @@ func CopyWorkflowExecutionInfo(sourceInfo *persistence.WorkflowExecutionInfo) *p
 		ParentDomainID:                     sourceInfo.ParentDomainID,
 		ParentWorkflowID:                   sourceInfo.ParentWorkflowID,
 		ParentRunID:                        sourceInfo.ParentRunID,
+		IsCron:                             sourceInfo.IsCron,
```

Comment:
bleh.  I really need to try to finish a "this struct must be completely filled out" linter...

Reply:
linter or unit test? 
one option could be to write tests using cmp.Diff and pass source object fully populated. to make it future proof, test can also validate all the public fields in the source object tree is non-nil.

Reply:
That's fundamentally what I'm looking to enable with the generator here - the filling out completely thing.